app/views/games/show.html.erb

```diff
@@ -87,11 +87,14 @@
 
 </div>
 
+<%
+related_games = @game.find_related_platforms.first(3)
```

Comment:
@JasonHoffmann if you extract that part into the controller (simply add: `@related_games = @game.find_related_platforms.first(3)`) and then use that `@related_games` variable in the view